.github/workflows/release-code.yml

```diff
@@ -11,16 +11,16 @@ jobs:
     if: github.repository == 'ExpediaGroup/graphql-kotlin'
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - uses: gradle/wrapper-validation-action@v1
 
       - name: Set up Java 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           java-version: 11
           distribution: 'zulu'
-
+      - uses: gradle/gradle-build-action@v2
```

Comment:
lets remove it -> when releasing the code we want to ensure whole thing can be build without relying on any caching

Reply:
5726ea9021bacf1ebea9b707e744ad7c4f9c47e2